statsmodels/stats/oneway.py

```diff
@@ -0,0 +1,272 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Mar 18 10:33:38 2020
+
+Author: Josef Perktold
+License: BSD-3
+
+"""
+
+import numpy as np
+from scipy import stats
+from scipy.special import ncfdtrinc
+
+from statsmodels.tools.testing import Holder
+
+
+def effectsize_oneway(means, vars_, nobs, use_var="unequal", ddof_between=0):
+    """effect size corresponding to Cohen's f^2 = nc / nobs for oneway anova
+
+    Parameters
+    ----------
+    means: array_like
+        Mean of samples to be compared
+    vars_ : float or array_like
+        Residual (within) variance of each sample or pooled
+        If var_ is scalar, then it is interpreted as pooled variance that is
+        the same for all samples, ``use_var`` will be ignored.
+        Otherwise, the variances are used depending on the ``use_var`` keyword.
+    nobs : int or array_like
+        Number of observations for the samples.
+        If nobs is scalar, then it is assumed that all samples have the same
+        number ``nobs`` of observation, i.e. a balanced sample case.
+        Otherwise, statistics will be weighted corresponding to nobs.
+    use_var : {"unequal", "equal"}
+        If ``use_var`` is "unequal", then the variances can differe across
+        samples and the effect size for Welch anova will be computed.
+    ddof_between : int
```

Comment:
I think `ddof_between` can be dropped.
(I might have interpreted things incorrectly when I rearranged the way things are computed)